stream-chat-android-state/src/main/java/io/getstream/chat/android/offline/plugin/logic/channel/thread/internal/ThreadLogic.kt

```diff
@@ -32,7 +32,7 @@ import io.getstream.chat.android.offline.plugin.state.channel.thread.internal.Th
 /** Logic class for thread state management. Implements [ThreadQueryListener] as listener for LLC requests. */
 internal class ThreadLogic(
     private val repos: RepositoryFacade,
-    private val threadStateLogic: ThreadStateLogic,
+    internal val threadStateLogic: ThreadStateLogic,
```

Comment:
Would you mind adding a specified method instead:
```kotlin
internal fun stateLogic(): ThreadStateLogic {
    return threadStateLogic
}
```

Just to be consistent and follow the same approach you've already applied in other `Logic` classes